src/Atmos/Parameterizations/CloudPhysics/Activation.jl

```diff
@@ -0,0 +1,154 @@
+
+
+Pkg.add("SpecialFunctions")
+module Activation
+
+using Thermodynamics
+using CLIMAParameters
+using CLIMAParameters.Atmos.Microphysics_0M
+const APS = AbstractParameterSet
+export activation_cloud_droplet_number
+# TODO: Check over these ^^^^
+
+"""
+    Isabella Dula and Shevali Kadakia
+    
+    This file has all the functions for the parametrization, according to the model
+    given in Abdul-Razzak and Ghan (2000).
+    
+"""
+#--------------------------------------------------------------------#
+# TODO: 
+# --Write function to determine mass fraction of soluble 
+#   material (epsilon)
+#--Determine P_saturation origins (do we get this from model?)
+#--Double check math syntax
+#--------------------------------------------------------------------#
+# CONSTANTS 
+
+WTR_MM = 0.01801528 # kg/mol
+WTR_MLR_ρ = 0.022414 # m^3/mol TODO (used for epsilon)
+WTR_ρ = 1000 # kg/m^3
+R = 8.31446261815324 # (kg m^2) / (s^2 K mol)
+AVOGADRO =  6.02214076 × 10^23 # particles/mole (used for epsilon)
+G = 9.81 # m/s^2
+LAT_HEAT_EVP = 2.26 * 10^-6  # J/kg
+SPC_HEAT_AIR = 1000
+T = 273.1 # K (STP)
+P = 100000 # Pa (N/m^2) (STP)
+
+#--------------------------------------------------------------------#
+# Critical supersaturation
+function S_m(activation_time, particle_radius, mass_mixing_ratio, 
+            disassociation, osmotic_coefficient, mass_ratio,
+            aerosol_molecular_mass, aerosol_density)
+    A = A(activation_time)
+    B_bar = mean_hygroscopicity(mass_mixing_ratio, disassociation, 
+            osmotic_coefficient, mass_ratio,
+            aerosol_molecular_mass, aerosol_density)
+    S_m = (2./(B_bar).^.5).*(A./(3.*particle_radius)).^(3/2)
+    return S_m
+end
+
+#--------------------------------------------------------------------#
+# Surface tension effects on Kohler equilibrium equation (s/(kg*m))
+function A(activation_time)
+    A = (2*activation_time.*WTR_MM)./(WTR_ρ.*R.*T)
+    return A
+end
+
+#--------------------------------------------------------------------#
+# Mean Hygroscopicity
+function mean_hygroscopicity(mass_mixing_ratio, disassociation, 
+                             osmotic_coefficient, mass_ratio,
+                             aerosol_molecular_mass, aerosol_density)
+    
+    B_bar = zeros(size(mass_mixing_ratio, 2))
```

Comment:
What is the `2` here? Is it assuming that there are two aerosol types? Should it be a parameter as well?

Reply:
I think that the 2 refers to the dimension over which you want the size of. With what we have now, each component is a 'row' in the aerosol array, and so I wanted to iterate over that dimension.